src/main/python/run_regression.py

```diff
@@ -53,15 +53,12 @@ def check_output(command):
 
 def get_index_path(yaml_data):
     """Find the index path."""
-    index_path = os.path.join('lucene-index.{0}.pos+docvectors{1}'.format(yaml_data['name'], \
-        '+rawdocs' if '-storeRawDocs' in yaml_data['index_options'] else ''))
-    if not os.path.exists(index_path):
-        index_path = yaml_data['index_path']
-        if not index_path or not os.path.exists(index_path):
-            for input_root in yaml_data['input_roots']:
-                if os.path.exists(os.path.join(input_root, yaml_data['index_path'])):
-                    index_path = os.path.join(input_root, yaml_data['index_path'])
-                    break
+    index_path = yaml_data['index_path']
+    if not index_path or not os.path.exists(index_path):
+        for input_root in yaml_data['input_roots']:
+            if os.path.exists(os.path.join(input_root, yaml_data['index_path'])):
+                index_path = os.path.join(input_root, yaml_data['index_path'])
```

Comment:
How about move this out of `if` and do checking `index_path`?

Reply:
yup, good call!